hashes/buildroot

```diff
@@ -16,8 +16,8 @@ SHA512 (gmp-6.2.1.tar.xz) = c99be0950a1d05a0297d65641dd35b75b74466f7bf03c9e8a998
 SHA512 (isl-0.25.tar.xz) = 81ac6b404a71e146bb705efe647ecf3bee19c3254f534cb44228cec13ffc7a33d7d58b980106dbb120ffdc557403d966619e219328edd0a4b3cbc4ac66acb255
 # https://musl.libc.org/releases/musl-1.2.3.tar.gz
 SHA512 (musl-1.2.3.tar.gz) = 9332f713d3eb7de4369bc0327d99252275ee52abf523ee34b894b24a387f67579787f7c72a46cf652e090cffdb0bc3719a4e7b84dca66890b6a37f12e8ad089c
-# https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-5.10.155.tar.xz
-SHA512 (linux-5.10.155.tar.xz) = fc763f7854ed5e8964fee2acc2f521caa215cbfa76dd7a054173fcf37f74ef543820e7b026c0ee2c01c178eb22c89ad334ec9559950772ec278d78d95b32d33c
+# https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-5.10.162.tar.xz
+SHA512 (linux-5.10.162.tar.xz) = 941c7ddd7c27f49b4491e1c8fbf1efedcbac50b48ed8836ec91091ead69723f820a61bfda795378dcc728a782d2206189903333e83e255b723eec01157bbb0bb
```

Comment:
Do we also want to upgrade `mpfr`?

Reply:
While there is one available, **Buildroot** is expecting a specific version of `mpfr`.